hw/ip_templates/otp_ctrl/dv/env/seq_lib/otp_ctrl_base_vseq.sv.tpl

```diff
@@ -155,10 +155,37 @@ class otp_ctrl_base_vseq extends cip_base_vseq #(
     super.read_and_check_all_csrs_after_reset();
   endtask
 
+  virtual task dai_zerorize(bit [TL_DW-1:0] addr);
+    bit [TL_DW-1:0] val;
+
+    `uvm_info(`gfn, $sformatf("dai Zerorize addr %0h", addr), UVM_HIGH)
+    csr_wr(ral.direct_access_address, addr);
+    csr_wr(ral.direct_access_wdata[0], '{default: 1});
+    if (is_secret(addr) || is_sw_digest(addr)) csr_wr(ral.direct_access_wdata[1], '{default: 1});
```

Comment:
Or if it is the address of a zeroized item (last item in a zeroizable partition).

Also, notice this assumes only secret partitions have a a hardware digest, which in OT is always the case.